float64 enc_az
float64 enc_el
